Neuron/Sections/Transaction/Details/TransactionDetailsViewController.swift

```diff
@@ -153,19 +153,19 @@ class TransactionDetailsViewController: UITableViewController {
 
 extension TransactionDetailsViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        if hiddenItems.contains(where: { $0.0 == indexPath.section && $0.1 == indexPath.row }) {
+        if paramBuilder == nil || hiddenItems.contains(where: { $0.0 == indexPath.section && $0.1 == indexPath.row }) {
             return 0.0
         } else {
             return super.tableView(tableView, heightForRowAt: indexPath)
         }
     }
 
     override func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
-        cell.isHidden = hiddenItems.contains(where: { $0.0 == indexPath.section && $0.1 == indexPath.row })
+        cell.isHidden = cell.bounds.height == 0.0
     }
 
     override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
-        return [(0, 1), (0, 2), (0, 3)].contains(where: { $0.0 == indexPath.section && $0.1 == indexPath.row })
+        return [(0, 1), (0, 2), (0, 3), (1, 0)].contains(where: { $0.0 == indexPath.section && $0.1 == indexPath.row })
```

Comment:
This has become too confusing and difficult to understand/maintain. We need to think of a better way to keep track of the showing/hidden rows.